sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2ScanExecBase.scala

```diff
@@ -35,7 +36,14 @@ trait DataSourceV2ScanExecBase extends LeafExecNode with ColumnarBatchScan {
   def readerFactory: PartitionReaderFactory
 
   override def simpleString(maxFields: Int): String = {
-    s"$nodeName${truncatedString(output, "[", ", ", "]", maxFields)} ${scan.description()}"
+    redact(s"$nodeName${truncatedString(output, "[", ", ", "]", maxFields)} ${scan.description()}")
```

Comment:
nit: seems we only need to redact `scan.description`?

Reply:
Well, I think redacting the whole string is safer.